### ISSUE TITLE
Support className passthrough

### ISSUE THREAD

**Issue by yanickrochon** [react-materialize/react-materialize#152]
User code should be able to set extra `className` values to just about any component.

* [x] `Button`
* [ ] `Breadcrumb` should set to the nav element
* [x] `Card`
* [x] `CardPanel`
* [ ] `CardTitle` should set to the top level div element
* [ ] `Chip` should set to the top level div element
* [x] `Col`
* [x] `Collapsible`
* [ ] `CollapsibleItem` should set to the li element
* [ ] `Collection` should set to the list (i.e. ul / div) element
* [x] `CollectionItem`
* [ ] `Dropdown` should set on the span (top level) element
* [x] `Footer`
* [ ] `Icon` should set on the i element
* [ ] `Input` should set to the input / textarea / select element
* [ ] `MediaBox` should set on the img element
* [x] `MenuItem`
* [ ] `Modal` should set to the div (top level) element
* [ ] `NavItem` should set to the li element
* [x] `Navbar`
* [x] `Pagination`
* [x] `PaginationButton`
* [ ] `Preloader` should set to the div (top level) element
* [x] `ProgressBar`
* [x] `Row`
* [ ] `SearchForm` should set to the form (top level) element
* [ ] `SideNav` should set to the nav (top level) element
* [x] `Slide`
* [ ] `Spinner` should set to the div (top level) element
* [ ] `Tab` should set to the top level element *(not implemented component)*
* [x] `Table`
* [x] `Tabs`
* [ ] `Tag` should pass to the `Chip` component
* [ ] `Toast` should set to the button element

*`Overlay` and `OverlayTrigger` are special cases... no?

**Comment by alextrastero** [comment]
Yes indeed @yanickrochon, I'm currently working on basic specs for each component, I'm also fixing the 0.15 warnings and I will also pass className props to the above mentioned, thanks for this!
